src/mcp_agent/workflows/llm/multipart_converter_anthropic.py

```
@@ -0,0 +1,508 @@
+            elif is_image_content(content_item):
+                # Handle image content
+                image_content = content_item  # type: ImageContent
+                # Check if image MIME type is supported
+                if not AnthropicConverter._is_supported_image_type(
+                    image_content.mimeType
+                ):
+                    data_size = len(image_content.data) if image_content.data else 0
+                    anthropic_blocks.append(
+                        TextBlockParam(
+                            type="text",
+                            text=f"Image with unsupported format '{image_content.mimeType}' ({data_size} bytes)",
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)